pkg/utils/utils.go

```diff
@@ -104,3 +104,21 @@ func GetEnvironmentVariables() (bool, string, string, string) {
 	}
 	return true, endpoint, user, password
 }
+
```

Comment:
the function is a bit strange, at least it requires an explanation. 
Is it correct?
the function transfers a map [string]interface{} to a map[sting]sting, with the given keys. If at least a one key is not in the given map, or its value is not a string, nil will be return.

Please add logs to the cases when you return nil.

Reply:
I am changing the name of the parameter from `fields` to `requiredFields`.
Adding documentations explaining that if any of the required fields are missing, `nil` is returned